apps/api/src/app/environments-v1/usecases/output-renderers/maily-to-liquid/wrap-maily-in-liquid.usecase.ts

```diff
@@ -157,7 +157,7 @@ export class WrapMailyInLiquidUseCase {
     node: MailyJSONContent
   ): node is MailyJSONContent & { attrs: { [MailyAttrsEnum.EACH_KEY]: string } } {
     return !!(
-      node.type === MailyContentTypeEnum.FOR &&
```

Comment:
duplication 🤭
for your consideration if we should extract and reuse here and on the renderer use case.

Reply:
on the same time we could rename to isRepeatNode